.github/workflows/scala-steward.yml

```diff
@@ -1,6 +1,6 @@
 on:
   schedule:
-    - cron: '0 0 1,15 * *'
+    - cron: "0 0 * * 1, 5" # every Monday and Friday
```

Comment:
I set Monday and Friday randomly (maybe we can run 3 times a week?)

Reply:
I think 2 is fine :+1:

Reply:
```suggestion
    - cron: "0 0 * * 1,5" # every Monday and Friday
```
I think there might an error and no space is allowed there?